webpack/scenes/ContentViews/Table/ContentViewsTable.js

```diff
@@ -26,15 +27,12 @@ const ContentViewTable = ({ response, status, error }) => {
   useEffect(
     () => {
       if (!loadingResponse && results) {
-        const { updatedRowMap, ...tableData } = tableDataGenerator(
-          results,
-          rowMapping,
-        );
+        const { newRowMappingIds, ...tableData } = tableDataGenerator(results);
         setTable(tableData);
-        setRowMapping(updatedRowMap);
+        setRowMappingIds(newRowMappingIds);
       }
     },
-    [results, JSON.stringify(rowMapping)], // use JSON to check obj values eq not reference eq
+    [results],
```

Comment:
Any variable that's not defined within the effect still needs to go in the dep array :)
```suggestion
    [results, loadingResponse, setTable, setRowMappingIds],
```

Reply:
Aah..That's right.. Updated.. 👍🏽 

So the current combination of compact table with padding for expanded children looks like this:

![Screenshot from 2021-04-28 09-34-36](https://user-images.githubusercontent.com/21146741/116413474-a4e97c00-a805-11eb-92c4-8e4d1092f7ce.png)

Reply:
I think that looks great!!